### PR TITLE
context added

### PR DESCRIPTION
**Pull request by NiclasNorin**

### CODE REVIEW

File: source/php/Module/Image/views/default.blade.php [hunk 1]
```diff
@@ -30,7 +30,9 @@
             'alt' => $mod_image_image['alt'],
             'caption' => (isset($mod_image_caption) && !empty($mod_image_caption)) ?
                 $mod_image_caption : "",
-            'classList' => ['block-level', $img_classes]
+            'classList' => ['block-level', $img_classes],
+            'context' => ['module.image', $sidebarContext . '.module.image', $sidebarContext . '.module.hero-animation'],
```
Comment (sebastianthulin) [inline source/php/Module/Image/views/default.blade.php:34]: Is "hero-animation" a relevant context name generally? Context should indicate where it is used or what it is. This describes what it should do.

File: source/php/Module/Slider/views/slider.blade.php [hunk 2]
```diff
@@ -22,7 +22,7 @@
         'data-slider-focus-center' => '',
         'data-aria-labels' => json_encode($ariaLabels)
     ],
-    'context'       => ['module.slider', $sidebarContext . '.module.slider'],
+    'context'       => ['module.slider', $sidebarContext . '.module.slider', $sidebarContext . '.module.hero-animation'],
```
Comment (sebastianthulin) [inline source/php/Module/Slider/views/slider.blade.php:25]: Same as above.

File: source/php/Module/Image/views/default.blade.php (hunk 1 above)
Comment (NiclasNorin) [reply source/php/Module/Image/views/default.blade.php:34]: Is hero-image a better name? 
It describes the item we want to target.

Comment (sebastianthulin) [reply source/php/Module/Image/views/default.blade.php:34]: Yes, but the sidebar context gives you this information i believe.

Comment (NiclasNorin) [reply source/php/Module/Image/views/default.blade.php:34]: That is true, maybe something like animated-item or animation-item?
It needs to fit different modules, "anmiation-item" could be used to identify items in other sidebar areas. Maybe that is a good name?

**Pull request merged**